packages/@react-aria/selection/src/useSelectableItem.ts

```diff
@@ -86,7 +86,13 @@ export function useSelectableItem(options: SelectableItemOptions): SelectableIte
     if (e.pointerType === 'keyboard' && isNonContiguousSelectionModifier(e)) {
       manager.toggleSelection(key);
     } else {
-      manager.select(key, e);
+      let eventType: SelectEventType = {};
+      if (e && e.shiftKey) {
+        eventType.extend = true;
+      } else if (e && (((e.metaKey && !isWindows()) || e.pointerType === 'touch' || e.pointerType === 'virtual') || (e.ctrlKey && isWindows()))) {
```

Comment:
Maybe move this util somewhere so it can be reused? https://github.com/adobe/react-spectrum/blob/a2815ee889920146f69075e9559db973c0a362d4/packages/%40react-aria/selection/src/useSelectableCollection.ts#L22

Reply:
Are you thinking of a util class/file with functions like isWindowsCtrlPress() and isMetaPress()? Similar to how we have isMac(), isIpad(), isWindows() in utils/platform.ts?

Reply:
I'd just move this function into `utils.ts` that's already in the selection package. Then it can be reused here rather than inlining the metaKey/ctrlKey logic.